<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hestia [*]</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabScenario">
       <attribute name="title">
        <string>Scenario</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitterScenario">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTreeView" name="treeViewScenario"/>
          <widget class="PlainTextEdit" name="textEditScenario"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabStructure">
       <attribute name="title">
        <string>Structure</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitterStructure">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTreeView" name="treeViewStructure"/>
          <widget class="QListView" name="listViewStructure"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabScript">
       <attribute name="title">
        <string>Script</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitterScript">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTreeView" name="treeViewScript"/>
          <widget class="QTextEdit" name="textEditScript"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionReloadProject"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuScenario">
    <property name="title">
     <string>Scenario</string>
    </property>
    <addaction name="actionScenarioNew"/>
    <addaction name="actionScenarioSave"/>
    <addaction name="actionScenarioSaveAs"/>
    <addaction name="actionScenarioRemove"/>
   </widget>
   <widget class="QMenu" name="menuScript">
    <property name="title">
     <string>Script</string>
    </property>
    <widget class="QMenu" name="menuTarget">
     <property name="title">
      <string>Target</string>
     </property>
     <addaction name="actionMarkdown"/>
     <addaction name="actionKAG3"/>
     <addaction name="actionNScripter"/>
    </widget>
    <addaction name="menuTarget"/>
   </widget>
   <widget class="QMenu" name="menuStructure">
    <property name="title">
     <string>Structure</string>
    </property>
    <addaction name="actionShowPreview"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuScenario"/>
   <addaction name="menuStructure"/>
   <addaction name="menuScript"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>SaveAs</string>
   </property>
  </action>
  <action name="actionScenarioNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionScenarioSaveAs">
   <property name="text">
    <string>SaveAs</string>
   </property>
  </action>
  <action name="actionScenarioRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionScenarioSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionMarkdown">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Markdown</string>
   </property>
  </action>
  <action name="actionKAG3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>KAG3</string>
   </property>
  </action>
  <action name="actionNScripter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>NScripter</string>
   </property>
  </action>
  <action name="actionShowPreview">
   <property name="text">
    <string>ShowPreview</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionReloadProject">
   <property name="text">
    <string>ReloadProject</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlainTextEdit</class>
   <extends>QTextEdit</extends>
   <header>widgets/plaintextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
